Snakefile

```diff
@@ -0,0 +1,104 @@
+import pandas as pd
+import os
+
+# create wildcard variables for workflow
+metadata = pd.read_csv("inputs/metadata.tsv", sep = "\t")   # read in metadata as a pandas dataframe
+TX = metadata['tx_run_accession'].unique().tolist()         # make run accession in tx col into list
+GX = metadata['gx_accession'].unique().tolist()             # make gx accession into list
+KSIZES = [21, 31, 51]                                       # create a list of k-mer sizes for the workflow
+gx_tmp = metadata['gx_accession'].tolist()
+TX_MINUS_GX = [x + '-minus-' + y for x, y in zip(TX, gx_tmp)]   # create list that binds tx to gx
+
+rule all:
+    input: 
+        expand("outputs/subtract_sourmash_sketch_describe/{tx_minus_gx}-k{ksize}.csv", tx_minus_gx = TX_MINUS_GX, ksize = KSIZES),
+        expand("outputs/gx_sourmash_sketch_describe/{gx_accession}.csv", gx_accession = GX),
+        expand("outputs/tx_sourmash_sketch_describe/{tx_run_accession}.csv", tx_run_accession = TX)
+
+rule download_doryteuthis_genome:
```

Comment:
Instead of the genomes being hard coded could the rule be `download_genomes` where it takes a separate csv/tsv file similar to your input metadata tsv that you pass in and it downloads based on those files provided? Just thinking about how to make this so others can input their genomes/SRAs of interest easily